test/integration/test_estimator.py

```diff
@@ -297,3 +297,30 @@ def test_estimator_error_messages(self, service):
                 job.result()
             self.assertIn("REGISTER NAME", str(err.exception))
             self.assertFalse("python -m uvicorn server.main" in str(err.exception))
+
+    @run_integration_test
+    def test_estimator_no_session(self, service):
+        """Test estimator primitive without a session."""
+
+        backend = service.backend(self.backend)
+
+        psi1 = RealAmplitudes(num_qubits=2, reps=2)
+
+        # pylint: disable=invalid-name
+        H1 = SparsePauliOp.from_list([("II", 1), ("IZ", 2), ("XI", 3)])
+
+        estimator = Estimator(backend=backend)
+        self.assertIsInstance(estimator, BaseEstimator)
+        self.assertIsNone(estimator.session)
+
+        theta1 = [0, 1, 1, 2, 3, 5]
+        circuits1 = [psi1]
```

Comment:
It seems to me you are testing only a single circuit. So why call it circuits1`? and why check the length of the results?

Reply:
> why call it circuits1

Because copy & paste 😄. I updated the code to avoid confusion. 

> why check the length of the results

See above.